### ISSUE TITLE
Warning: On Windows, creating symlinks requires administrator privileges

### ISSUE THREAD

**Issue by sschuldenzucker** [JuliaLang/PackageCompiler.jl#249]
I just compiled Plots. Whenever I start julia with the compiled system image, I get the following warning:

```
┌ Warning: On Windows, creating file symlinks requires Administrator privileges
└ @ Base.Filesystem file.jl:794
```

(I'm indeed on Windows 10)

The warning does not occur when I use the original system image.

What is going on here? Why is julia trying to create a symlink? Is it dangerous?

**Comment by SimonDanisch** [comment]
That's likely, because BinaryProvider ended up being compiled into your system image..
BinaryProvider prints this warning on Windows when used... Although on a newer version it shouldn't do this anymore!

**Comment by sschuldenzucker** [comment]
All I did was `compile_incremental(:Plots, install=true)` on the `sd-notomls` branch on dev (as of #239). It must've been PackageCompiler that pulled in that dependency. `] st` indeed tells me that I got BinaryProvider v0.5.4 installed. This also seems to be the most recent stable version.

**Comment by arnavs** [comment]
Yes, the commit which fixed this error on Windows hasn't been merged yet.

**Comment by KristofferC** [comment]
Not a PackageCompiler issue.
